tests/e2e/config/config.go

```diff
@@ -109,6 +109,10 @@ func NewDefaultOperatorConfig(cfg *tests.Config) *tests.OperatorConfig {
 		PodWebhookEnabled:         false,
 		Cabundle:                  "",
 		BackupImage:               cfg.BackupImage,
+		StringValues: map[string]string{
+			"admissionWebhook.failurePolicy.validation": "Fail",
+			"admissionWebhook.failurePolicy.mutation":   "Fail",
+		},
```

Comment:
we will try to kill tidb-operator pods in e2e. we should fail API requests otherwise some tests will fail if the admission webhook is absent.